matter/efr32/efr32mg24/BRD4187A/autogen/sl_device_init_clocks.c

```diff
@@ -33,12 +33,12 @@
 
 sl_status_t sl_device_init_clocks(void)
 {
-    CMU_ClockSelectSet(cmuClock_SYSCLK, cmuSelect_HFXO);
+    CMU_ClockSelectSet(cmuClock_SYSCLK, cmuSelect_HFRCODPLL);
```

Comment:
Like brd4186A, the swap to HFRCODPLL seems odd

Reply:
when adding the dpll componenent, clock value is set to that